ui/apps/platform/src/Containers/VulnMgmt/Entity/Cve/VulnMgmtEntityCve.js

```diff
@@ -35,9 +42,20 @@ const vulnFieldMap = {
     CLUSTER_CVE: CLUSTER_CVE_DETAIL_FRAGMENT,
 };
 
+function getCVETypeFromStack(worklowStateStack) {
```

Comment:
"CVET" looks like the name of a Cold War-era paramilitary group.

Gen X, ftw ;)